src/fieldset/index.jsx

```diff
@@ -32,6 +41,13 @@ class Fieldset extends Component {
     this.onFieldChange = this.onFieldChange.bind(this);
   }
 
+  static getDerivedStateFromProps(nextProps, prevState) {
```

Comment:
hmm, this feels like an odd lifecycle method to use - could we swap for `componentDidReceiveProps?`

Reply:
also I don't think it need to be static

Reply:
actually ignore me  - I'm talking rubbish again

Reply:
the static part can be removed though :)

Reply:
the internet says use `componentDidUpdate`